src/utils/thread_helper.cpp

```diff
@@ -920,6 +920,43 @@ void ThreadHelper::DetachGracefullyCallback(void)
 }
 
 #if OTBR_ENABLE_TELEMETRY_DATA_API
+#if OTBR_ENABLE_BORDER_ROUTING
+void ThreadHelper::GetNodeDivergenceInfo(threadnetwork::TelemetryData_NodeDivergenceInfo *aNodeDivergenceInfo)
+{
+    bool      isExternalRoutePublished = false;
```

Comment:
There is a difference between "publish" and "add" terms used within OT regarding Network Data. I mentioed more details here https://github.com/openthread/openthread/pull/10199#discussion_r1591346486

Suggest renaming to `isExternalRouteAdded` as this is what is being checked below.

Reply:
Thanks! Updated!